### DISCUSSION TITLE
Display Issue/Question

### DISCUSSION THREAD

**Discussion by kfishermail** [j-kaltes/Juggluco#88] [General]
I activated a new sensor today, and I noticed something.
Typically the there is a graph on the left 2/3 of the screen,
and a text display of the current glucose on the right side
of the screen.  This text value is normally centered vertically
on the screen.  Now it is located closer to the top of the 
screen (i.e. the bottom of the text is now where the top used to be)

I am doing a screen capture, followed by some image processing
and need the location to be fixed.

Any idea as to why the location has changed.

**Comment by j-kaltes** [comment]
If Juggluco uses two sensors, they are spaced differently. You can terminate one of them by selecting the sensor in Left menu->Sensor and pressing _Terminate_.
